### DISCUSSION TITLE
2023-10-12

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#638] [daily_meeting]
### 오늘의 할 일

- 카카오 추가 버전 dev 배포

### 내일의 할 일

- 폴링 버전 dev 배포

**Comment by nlom0218** [comment]
### 체크인

9점 - 상쾌하지만 살짝 눈이 무거운 느낌이네요

**Comment by jaehee329** [comment]
### 체크인
7점 - 문제 풀기 실패했어요

**Comment by woo-jk** [comment]
### 체크인
8점 - 오늘도 역시 피곤해요

**Comment by woosung1223** [comment]
### 체크인
8점 - 벌써 목요일이군요.. ㄷㄷ

### 체크아웃
8점 - 오늘 초밥 먹을 생각에 좋네여

**Comment by aak2075** [comment]
### 체크인

10점 - 문제가 어렵네요ㅠ

### 체크아웃

10점 - 시간이 너무 빨라요

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 할게 넘쳐나네요. 살아있음을 절실하게 체감하고 있습니다.

### 체크아웃
5점 - 인적성 검사 만만하게 볼게 아니네요

**Comment by yeopto** [comment]
### 체크인
7점 - 감기기운 쉽지않네요

### 체크아웃
5점 - 약기운에 헤롱헤롱

**Reply by nlom0218** [reply]
힘내십쇼 건또건입니다

**Reply by aak2075** [reply]
건또건ㅋㅋ
